### PR TITLE
Improves (csv-) bulk-importers

### LINKED ISSUE #92: Bulk actions
Finish #69

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
Fixes #92

### CODE REVIEW

File: client/src/app/core/ui-services/base-import.service.ts [hunk 1]
```diff
@@ -381,4 +358,150 @@ export abstract class BaseImportService<M extends BaseModel> {
...
+     * by submitting them to the server. The entries will receive the status
+     * 'done' on success.
+     */
+    public async doImport(): Promise<void> {
+        const promises: Promise<any>[] = Object.values(this.importHelperMap).map(helper =>
+            helper.createUnresolvedEntries()
+        );
+        await Promise.all(promises);
+
+        const newUsers: any[] = [];
```
Comment (FinnStutzenstein) [inline client/src/app/core/ui-services/base-import.service.ts:374]: Why is it called `newUsers` in a base import class?

File: client/src/app/site/agenda/components/list-of-speakers/list-of-speakers.component.ts [hunk 2]
```diff
@@ -211,4 +214,91 @@ export class ListOfSpeakersComponent extends BaseModelContextComponent implement
...
+    }
+
+    /**
+     * get the duration of a speech
+     *
+     * @param speaker
+     * @returns string representation of the duration in `[MM]M:SS minutes` format
+     */
+    public durationString(speaker: ViewSpeaker): string {
+        throw new Error('TODO!');
```
Comment (FinnStutzenstein) [inline client/src/app/site/agenda/components/list-of-speakers/list-of-speakers.component.ts:235]: todo

File: client/src/app/site/motions/modules/motion-import/motion-import-list.component.html [hunk 3]
```diff
@@ -25,7 +25,7 @@ <h2>{{ 'Import motions' | translate }}</h2>
     <ul>
         <li>
             {{
-                'Number, reason, submitter, category, origin and motion block are optional and may be empty.'
+                'Identifier, reason, submitter, category, origin and motion block are optional and may be empty.'
```
Comment (FinnStutzenstein) [inline client/src/app/site/motions/modules/motion-import/motion-import-list.component.html:28]: Number is correct - identifier is the OS3 term

**Review by FinnStutzenstein — APPROVED** [review]
Looks good. I did not test it and looked more briefly. But I like the enhanced structure in the base import service. Some minor remarks.

File: client/src/app/site/motions/modules/motion-import/motion-import-list.component.html (hunk 3 above)
Comment (GabrielInTheWorld) [reply client/src/app/site/motions/modules/motion-import/motion-import-list.component.html:28]: Ah, okay :+1:

**Pull request merged**